src/lib/web-component/filter-container/filter-container-element.js

```diff
@@ -0,0 +1,90 @@
+class FilterContainer extends HTMLElement {
+  constructor() {
+    super();
+
+    this.checkboxes = new Set([]);
+
+    const shadow = this.attachShadow({ mode: 'open' });
+
+    this.checkboxTemplate = document.querySelector(
+      '#filterCheckboxTemplate'
+    ).content;
+
+    const filterTemplate = document
```

Comment:
Can we re-name the web component as `filter-option`? since it's making new filtering option to filter logs in the table. Also every time we are re-using the component, we are making a new filter option to filter the logs in the table.

Reply:
How about renaming the web component as `filter-box` ?

Reply:
`filter-option` sounds more descriptive than `filter-container` or `filter-box`. I like that you're thinking of a better name for the custom element on your own.